src/ray/common/status.cc

```diff
@@ -59,6 +59,54 @@ namespace ray {
 #define STATUS_CODE_OBJECT_STORE_FULL "ObjectStoreFull"
 #define STATUS_CODE_TRANSIENT_OBJECT_STORE_FULL "TransientObjectStoreFull"
 
+static std::map<StatusCode, std::string> code_to_str = {
```

Comment:
no problem

Reply:
I changed the data type to flat_hash_map but keep it in ray namespace, as `StatusCode` is defined in ray namespace so it's natural.

Reply:
There seems to be no reason to put `code_to_str` and `str_to_code` in the file scope. Let's just keep them inside the functions where they are used.

Side node: for file scope variables, Ray tries to follow google c++ style guide which mentions [`use of internal linkage in .cc files is encouraged for all code that does not need to be referenced elsewhere`](https://google.github.io/styleguide/cppguide.html#Internal_Linkage). Common names like `ray::code_to_str` and `ray::str_to_code` are easy to have accidental ODR violations, if they have external linkages.

Reply:
ok to me. I put them in the file scope just to make them staying close with each other, so it would be harder to miss one of them when a new code added.
I'll put some comments there.